channels/generic/websocket.py

```diff
@@ -44,11 +44,16 @@ def websocket_connect(self, message):
     def connect(self):
         self.accept()
 
-    def accept(self, subprotocol=None):
+    def accept(self, subprotocol=None, headers=None):
         """
         Accepts an incoming socket
         """
-        super().send({"type": "websocket.accept", "subprotocol": subprotocol})
+        message = {"type": "websocket.accept", "subprotocol": subprotocol}
+        if self.scope.get("spec_version", "2.0") >= "2.1":
```

Comment:
Is there a need for this kind of conditional?

Reply:
I don't know, not sure if WSGI allows unspecified extra data in its messages.  A server with an older version will _probably_ just ignore it, but I wasn't sure.  What is the usual practice, do you think?

Reply:
Good hustle on the asgiref ticket @kristjanvalur. Looks like we can drop this check? 

If you do that I'll pull this in. 

Thanks! 🎁